docs/contribute/index.md

```diff
@@ -10,20 +10,114 @@ Whether you're a technical writer or you've only edited Elastic docs once or twi
 
-### Contribute to Elastic Stack version 8.x docs and earlier
+### Contribute to version `8.x` docs and earlier
 
-To contribute to earlier versions of the Elastic Stack, you must work with our [legacy documentation build system](https://github.com/elastic/docs). This system uses AsciiDoc as it's authoring format.
+To contribute to earlier versions of the Elastic Stack, you must work with our [legacy documentation build system](https://github.com/elastic/docs). This system uses the [AsciiDoc](https://asciidoc.org) markup language. 
 
 * For **simple bugfixes and enhancements** --> [Contribute on the web](on-the-web.md)
 * For **complex or multi-page updates** --> See the [legacy documentation build guide](https://github.com/elastic/docs?tab=readme-ov-file#building-documentation)
 
-### Contribute to Elastic Stack version 9.0 docs and later
+If you need to update documentation for both 8.x and 9.x, you will have to work in two systems. See [What if I need to update both 8.x and 9.x docs?](https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/contribute/on-the-web#what-if-i-need-to-update-both-8.x-and-9.x-docs) for the processes on updating docs across different versions.
+
+### Contribute to version `9.0` docs and later
+
+To contribute to `9.0+` content, you need to work with our new documentation system. This system uses custom [Markdown syntax](../syntax/index.md).
+This content lives in one of three places:
+
+1. **Reference/settings content** lives in the new Markdown files in the code repo as it did pre 9.0. This enables folks to make updates that touch code and low-level docs in the same PR.  
+2. **Narrative/conceptual and overview content** lives in [elastic/docs-content](https://github.com/elastic/docs-content). For example, the [ES|QL overview](https://docs-v3-preview.elastic.dev/elastic/docs-content/tree/main/explore-analyze/query-filter/languages/esql) lives in the **Explore & Analyze** section of the narrative docs. But the reference documentation lives in the Elasticsearch reference section of the docs: [ES|QL reference](https://docs-v3-preview.elastic.dev/elastic/elasticsearch/tree/main/reference/query-languages/esql)  
+3. **API reference docs** live in the different OpenAPI spec repositories. This is where you need to update API docs published in the new API docs system.
+
+When you are ready to edit the content:
 
 * For **simple bugfixes and enhancements** --> [contribute on the web](on-the-web.md)
 * For **complex or multi-page updates** --> [Contribute locally](locally.md)
 
+## Where to find docs source files
+
+Understand where your content of interest now sits in the new docs information architecture and the repos where source pages live.
```

Comment:
I am concerned about having this list be so precise and simultaneously incomplete. we're covering 1/20 of the content

Reply:
I think we should focus on explaining the shape of the changes rather than explaining what in particular has moved where (closer to what you have below)

remember that redirects and mapping will help with this